backend/schemas/source/base/Base.libsonnet

```diff
@@ -383,15 +383,15 @@ local type_uei = Types.string {
     // Does not start with 9 digits in a row
     {
       pattern: '^(?![0-9]{9})',
-    },
+    }
   ],
 };
 
 local Compound = {
   AwardReference: Types.string {
     title: 'AwardReference',
     description: 'Award Reference',
-    pattern: '^AWARD-(?!0000)[0-9]{4}$',
```

Comment:
These commas are automatically added by the JSONnet formatting tool when you run `jsonnetfmt -i <pathToYourFile>`.